ckanext/fairdatapoint/profiles.py

```diff
@@ -18,12 +18,16 @@ def _convert_extras_to_declared_schema_fields(dataset_dict):
     data_dict = {'type': dataset_type}
     full_schema_dict = toolkit.get_action('scheming_dataset_schema_show')(context, data_dict)
 
-    dataset_field_list = [x.get('field_name') for x in full_schema_dict.get('dataset_fields', [])]
+    dataset_field_list = { x.get('field_name') : x.get('preset') for x in full_schema_dict.get('dataset_fields', []) }
```

Comment:
please consider renaming so variable name reflects the structure

Reply:
Actually there is no need to add the collection type to the variable name. I will rename it to dataset_fields.